app/controllers/snapshots_controller.rb

```diff
@@ -1,14 +1,9 @@
 class SnapshotsController < ApplicationController
-
 	before_filter :make_api_public, only: [:index]
-
 	respond_to :json
 
 	def index
-
 		@snapshots = Snapshot.where(:keyframe => true).order(created_at: :desc)
-		render json: {snapshots: @snapshots}
-
+		render json: @snapshots
```

Comment:
FYI, `ActiveModel::Serializers` picks up on this and serializes automatically (and with a root key, I think). However, this should be tested :D